modules/social_features/social_comment/social_comment.module

```diff
@@ -291,26 +291,10 @@ function social_comment_comment_view(array &$build, EntityInterface $entity, Ent
     if (is_object($commented_entity)) {
       $commented_entity_type = $commented_entity->getEntityTypeId();
 
-      // Define which display mode should be used for commented_entity.
```

Comment:
The logic was moved to the `social_challange` module, so we don't need these conditions here anymore.